Source/RapyutaSimulationPlugins/Private/Tools/LimitRTFFixedSizeCustomTimeStep.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
+
+#include "Tools/LimitRTFFixedSizeCustomTimeStep.h"
+
+#include "Misc/App.h"
+#include "Misc/ConfigCacheIni.h"
+
+ULimitRTFFixedSizeCustomTimeStep::ULimitRTFFixedSizeCustomTimeStep(const FObjectInitializer& ObjectInitializer)
+    : Super(ObjectInitializer), StepSize(0.01), TargetRTF(1)

```

Comment:
```suggestion
    : Super(ObjectInitializer)
```
for already defined in class header